Sources/PT.PM.Matching.Tests/JavaMatchingTests.cs

```diff
@@ -40,6 +40,12 @@ public void Match_TestPatternsJava_MatchedAllDefault()
                 Assert.Greater(matchingResults.Count(p => p.PatternKey == dto.Key), 0, dto.Description);
             }
 
+            var ImproperValidationEmptyMethod_partial = patternDtos.Single(dto => dto.Description.StartsWith("ImproperValidationEmptyMethod_partial"));
```

Comment:
Local vars starts from lower-case char and does not have underscores (common code style).